UnityProject/Assets/ScriptableObjects/Speech/CustomMods/SlurredMod.cs

```diff
@@ -13,12 +13,14 @@ private void DoEffectTimeCheck()
 	{
 		if (drunkSpeechTime > 0)
 		{
-			playerMind.inventorySpeechModifiers |= ChatModifier.Drunk;
+			//This is it used anywhere???
+			//playerMind.inventorySpeechModifiers |= ChatModifier.Drunk;
```

Comment:
Unhelpful comment and don't comment out code.

Also giving a quick glance at this; this seems to be related to the drunk effect that is missing from the game. Would be good if you reimplemented it back again instead of slowly erasing it more.

Reply:
it's not implemented anywhere,